app/src/main/kotlin/com/wire/android/ui/conversation/model/Conversation.kt

```diff
@@ -0,0 +1,27 @@
+package com.wire.android.ui.conversation.model
+
+data class Conversation(
+    val userInfo: UserInfo,
+    val conversationInfo: ConversationInfo
+)
+
+data class ConversationInfo(
+    val name: String,
+    val memberShip: Membership = Membership.None,
+    val isLegalHold: Boolean = false
+)
+
+data class UserInfo(
+    val avatarUrl: String = "",
+    val availabilityStatus: AvailabilityStatus = AvailabilityStatus.None
+)
+
+enum class AvailabilityStatus {
+    Available, Busy, Away, None
+}
+
+enum class Membership(val label: String) {
+    Quest("Quest"), External("External"), None("")
```

Comment:
`Guest`, not `Quest`.


Also, about Internationalization.
Instead of holding a reference to a String (`label`), can we hold a reference to a String resource? Something like `R.string.converesation_membership_guest` and `R.string.conversation_membership_external`?
And instead of `None`, `ConversationInfo.membership` could be actually nullable. At some point we need to check if `membership != None`. So we might as well do `membership != null`, which is kinda easier to understand that this one is not special.